docs/tables/aws_guardduty_finding/index.md

```diff
@@ -0,0 +1,224 @@
+---
```

Comment:
For doc updates, does the S3 bucket source doc also need an update for defaults?